launch_ros/launch_ros/utilities/parse_composable_node.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module containing parser for composable_node tag."""
+
+from typing import List
+
+from launch.frontend import Entity
+from launch.frontend import Parser
+from launch_ros.actions.node import Node
+from launch_ros.descriptions import ComposableNode
+
+
+def parse_composable_node(parser: Parser, entity: Entity):
```

Comment:
@kenji-miyake any reason not to make this function a class method of `ComposableNode`?

Reply:
@hidmic No reason. I just didn't know where to put this function. Will make a class method!

Reply:
When I write `from launch_ros.actions import Node` (or `from launch_ros.actions.node import Node`) in `descriptions/composable_node.py`, I got an error. How can I solve this?

```
ImportError: cannot import name 'Node' from partially initialized module 'launch_ros.actions' (most likely due to a circular import) (/home/kenji/launch_test_ws/build/launch_ros/launch_ros/actions/__init__.py)
```

Reply:
Thank you, fixed in c119e94.